### ISSUE TITLE
Minor bug for quest Preparation for Ceremony

### ISSUE THREAD

**Issue by PierrickGT** [AeroScripts/QuestieDev#172]
Questie doesn't inform the player that you can loot [Bronze Feather] (http://www.wowhead.com/item=4753/bronze-feather#dropped-by) on [Winfury Matriarch] (http://www.wowhead.com/npc=2965/windfury-matriarch) for the quest [Preparation for Ceremony] (http://www.wowhead.com/quest=744/preparation-for-ceremony).

**Comment by Dyaxler** [comment]
Azure Feather is there but you're right... Bronze Feather is missing from the DB. I know how to get it in there but the item iterations are specific to Questie and aren't in order numerically. There are over 10K items in there and I have no idea which one is the last item in the list. Without knowing that I wouldn't know which number to assign to Bronze Feather. :) Perhaps one of the other contributors can point me to where these items might be listed numerically so we'll know which number to assign it to. If I were an Excel wizard I could import the table and create a custom pivot that would list the items for me numerically but sadly my Excel skills suck.

**Comment by Schaka** [comment]
`["Azure Feather"] = {
    ["drop"] = {
      ["Windfury Sorceress"] = 1586
    }
  },`

Similarly, just create a new entry (order doesn't matter, only hash access is used for tables) with the info of which mob drops the required item. The number itself is arbitrary and leftover from the QuestHelper DB. It used to represent how many drops were recorded with the QuestHelper addon.

Monsters.lua also has some information on drops, as we were in the process of refactoring the DB entirely and replacing QuestHelper leftovers. I'm not sure actually, which one of the two needs to the information. So don't worry about data redundancy.

**Comment by Dyaxler** [comment]
Yeah, all that was pretty easy to figure out in terms of where to put all the individual entries. So to be clear, the hash numbers aren't unique? For example if I were to add...

`["Bronze Feather"] = { ["drop"] = { ["Windfury Sorceress"] = 1587 } },`

Item 1587 already exists as...

```
  ["Carapace de tortue durcie"] = {
    ["drop"] = {
      ["Mordeuse nacrétoile"] = 1587,
      ["Tortue nacrétoile"] = 1759,
      ["Tortue fouisseuse nacrétoile"] = 445
    }
  },
```
I want to make sure I don't break anything. :)

**Comment by Schaka** [comment]
Yep, the numbers are arbitrary and aren't actually used ANYWHERE. But as it goes, you have to assign SOME value in Lua, not just the key. So the old QuestHelper DB used the amount of times the item was found/mob was seen when running the addon on retail back in the day to collect data for the database.

**Comment by Dyaxler** [comment]
Entries added - will be fixed in next release.

**Comment by Dyaxler** [comment]
Fixed in v2.3
